File: Baka.py
from flask import Flask, send_from_directory
import os

BASE = os.path.dirname(os.path.abspath(__file__))
HTML_DIR = os.path.join(BASE, "html")
CSS_DIR = os.path.join(BASE, "css")
JS_DIR = os.path.join(BASE, "js")

app = Flask(__name__)

@app.route("/")
def home():
    return send_from_directory(HTML_DIR, "cart.html")
@app.route("/<path:filename>")
def serve_html(filename):
    return send_from_directory(HTML_DIR, filename)

@app.route("/css/<path:filename>")
def serve_css(filename): 
    return send_from_directory(CSS_DIR, filename)

@app.route("/js/<path:filename>")
def serve_js(filename):
    return send_from_directory(JS_DIR, filename)

if __name__ == "__main__":
    port = int(os.environ.get("PORT", 8080))
    app.run(host="0.0.0.0", port=port)

File: Baka.js
const express = require("express")
const path = require("path")

const app = express()

// Base directory
const BASE = __dirname

// Directories
const HTML_DIR = path.join(BASE, "html")
const CSS_DIR = path.join(BASE, "css")
const JS_DIR = path.join(BASE, "js")

// 🔹 Home route → menu.html
app.get("/", (req, res) => {
  res.sendFile(path.join(HTML_DIR, "menu.html"));
})

// 🔹 Serve any HTML file
app.get("/:filename", (req, res) => {
  res.sendFile(path.join(HTML_DIR, req.params.filename));
})

// 🔹 Serve CSS
app.use("/css", express.static(CSS_DIR))

// 🔹 Serve JS
app.use("/js", express.static(JS_DIR))

// 🔹 Start server
const PORT = process.env.PORT || 8080
app.listen(PORT, "0.0.0.0", () => {
  console.log(`Server running on port ${PORT}`);
});